### ISSUE TITLE
[syncfusion_flutter_pdfviewer] PDF not rendering on landscape ,

### ISSUE THREAD

**Issue by Joseph-Nathan** [syncfusion/flutter-widgets#1206]
when i open pdf file  on Tablet devices on Landscape mode ,
the file not rendering ,
if i change the device to PortraitUp mode , 
and try opening the page again , the file rendering well ,

i test it on [ samsung SM-T585       8.1.0 ]
i have +30 user report this issues .

**Comment by deepika134** [comment]
We tried to reproduce the reported issue with provided details, but the PDF is loaded properly on our side. For your reference, we have prepared a simple sample (along with the output scree record of the same) and it can be downloaded from the attachment.

If you still reproduce the reported issue in your use case, then kindly share the modified issue reproducing in the sample, share the issue replicating video and flutter doctor log. Which will be helpful for us to provide a prompt solution at the earliest.
[github_466445.zip](https://github.com/syncfusion/flutter-widgets/files/11550541/github_466445.zip)
